src/pages/howManyProducts.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { ReactElement } from 'react';
+import { NextPageContext } from 'next';
+import { parseCookies } from 'nookies';
+import Layout from '../layout/Layout';
+import { NUMBER_OF_PRODUCTS_COOKIE, CSV_ZONE_UPLOAD_COOKIE } from '../constants';
+import { deleteCookieOnServerSide, buildTitle } from '../utils';
+import ErrorSummary from '../components/ErrorSummary';
+import { ErrorInfo } from '../types';
+
+const title = 'How Many Products - Fares data build tool';
+const description = 'How many products page of the Fares data build tool';
+
+export type InputCheck = {
```

Comment:
change to interface as discussed to keep consistency.

Reply:
Done!